README.md

```diff
@@ -89,10 +91,10 @@ var device = new Device({
 * `key`: The Losant access key.
 * `secret`: The Losant access secret.
 * `transport`: The underlying transport mechanism. Supports `tcp`, `tls`, `ws` (WebSocket), and `wss` (Secure WebSocket). Optional. Defaults to `tls`.
+* `qosPublish`: The QoS level to use for publishing messages. Defaults to 0.
```

Comment:
Would probably recommend adding only 0 and 1 are supported. 2 (exactly once) is not supported.